python_course_1604/class_01/tests/test_04.py

```diff
@@ -12,6 +14,9 @@
 
 
 @skip_if_exercize_not_started(exercize)
+@unittest.skipIf(
```

Comment:
This is not working:
```
E   NameError: name 'socket' is not defined
```
That means you have to import socket module at the top of the exercise:

```Python
import socket
```